packages/core/useVirtualList/index.ts

```diff
@@ -194,7 +198,7 @@ function createGetDistance<T>(itemSize: UseVirtualListItemSize, source: UseVirtu
     }
 
     const size = source.value
-      .slice(0, index)
+      .slice(0, index + 1)
```

Comment:
Slice end index is exclusive. Without this and with 0 overscan, we never render the end of the list.